app/models/custom_organisation_name.rb

```diff
@@ -1,3 +1,16 @@
 class CustomOrganisationName < ApplicationRecord
-  validates :name, presence: true, uniqueness: { case_sensitive: false }
+  validate :validate_in_register?
```

Comment:
Wonder if it's redundant to have the validate method have "validate" in its name.

Reply:
Change the method name to 'custom_org_in_register?'